CLOUDSTACK-8906: /var/log/cloud/ doesn't get logrotated on xenserver

SudharmaJain opened the pull request:
After integrating XS with CCP the following folder gets created: /var/log/cloud/ however the logs in that are not rotated resulting in root file system fill up. It was a known issue and link http://support.citrix.com/article/CTX138064 describes the issue and solution. Used the article and added corresponding changes to Cloudstack.

remibergsma commented:
@SudharmaJain The title and the content of your PR do not match. You seem to re-implement the XenServer 6.0.2 resource? If that is what you want, make a PR just for that. Also I'd like to know if the logrotate you make works on 6.2 and 6.5 as well.

Did you also see PR #861 on this subject?

SudharmaJain commented:
@remibergsma I didn't checked the PR #861. I was looking to fix the issue with xenserver 6.0.2 and higher. For this I have added a logrotate script and a Patch file for 6.0.2 which is the same as for xenserver 6.0 with an extra entry for logrotate script. same entry is added in patch file for 6.2 and 6.5. In earlier version, you can find logrotate in /etc/cron.hourly. In later versions it was not available which broke the log rotate functionality.

remibergsma commented:
@SudharmaJain Can you please rebase against current master? Then we can run the tests.

SudharmaJain commented:
@remibergsma Rebased against the current master.

koushik-das commented:
@SudharmaJain Why can't the XenServer600Resource be used to handle XS 6.0.2 hosts? The only change I see is the patch script (scripts/vm/hypervisor/xenserver/xenserver602/patch). If the content of the patch file is same as that of scripts/vm/hypervisor/xenserver/xenserver60/patch then there is no need for a separate 6.0.2 resource, PR #861 should be used instead.

remibergsma commented:
@SudharmaJain Please respond to comments. If you agree, please close this PR. Thanks!

SudharmaJain commented:
@koushik-das This bug exist in xenserver 6.0.2 and later. For XenServer 6.0, logrotation works. Thats is the reason i have implemented XenServer602Resource and didn't used Xenserver600Resource. Contents of patch file for 6.0 and 6.0.2 had a difference of one line. I have added a new file logrotate to be deployed in /etc/cron.hourly for versions 6.0.2 and later. So i have updated the patch file for 6.2 and 6.5 as well.

SudharmaJain commented:
@remibergsma  @koushik-das For incorporating the change in XenServer600Resource, I need to test the change for Xenserver 6.0. After checking it for 6.0, I will update the PR.

remibergsma commented:
@SudharmaJain Please test PR #861 and confirm your problem has been solved. We'll then close this PR and merge the other.

bhaisaab commented:
LGTM (code-wise), let's merge this once tested.

remibergsma commented:
We won't merge this PR, we should merge #861 instead. Final call to @SudharmaJain to confirm if PR #861 also fixes your issue. If I do not get a response in the coming days, I will merge #861 and close this PR. See for more info the discussion in #861.

SudharmaJain commented:
@remibergsma @koushik-das I have made the changes and have removed reimplementation of XenServer 6.0.2 resource. I have added changes to patch file for XenServer 6.0 resource, so that log rotation will work for 6.0 and 6.0.2.  PR #861 will not fix this issue. Now both PR #883 and PR #861 need to be merged to fix the issues.

remibergsma commented:
@koushik-das I merged #861 and @SudharmaJain updated this PR and removed the reimplementation of XenServer 6.0.2 resource. Are you OK with this change now? Please review again, thanks!

remibergsma commented:
@koushik-das Any update on this please? Thanks!

remibergsma commented:
@sspans Can you review this xenserver logrotate change please? Thanks!

koushik-das commented:
Changes LGTM. I haven't tested it.

sspans commented on scripts/vm/hypervisor/xenserver/logrotate:
```diff
@@ -0,0 +1,29 @@
+#
+# script to perform logrotation on xenserver 6.0.2 and later
+
+/usr/sbin/logrotate  /etc/logrotate.d/cloudlog
```
Should this really be only the cloudlog config?

SudharmaJain replied:
@sspans  I think the files mentioned in the patch file are the only ones getting deployed to the xenserver. Of these I cloud find cloudlog is the only logrotate configuration file. 
Please let me know if I missed anything.

remibergsma commented:
Hi @SudharmaJain can you please fix the conflict? I'll then run some tests and we're ready to merge.

bhaisaab commented:
@SudharmaJain Please rebase and send for a re-review

SudharmaJain commented:
@bhaisaab Rebased the branch.

bhaisaab commented:
@SudharmaJain Will @swill is the RM for master/4.9, I'm not following these changes perhaps any Xen guru can comment, @agneya2001 ?
@remibergsma do you have any outstanding issues with this PR

swill commented:
I will admit, I don't really understand everything in play here.  If I read this quickly, it seems like this PR may not be needed because #861 essentially changes the database to point to the `XenServer600Resource` resource which does not have the logrotate problem.  Can I get a clarification from @remibergsma and/or @SudharmaJain.  Thanks...

SudharmaJain commented:
The existing implementation for logrotation that works with xenserver 6.0 and earlier does not works with 6.0.2 and later. That is the reason I have added this implentation. With #861 we are changing the database to point to XenServer600Resource, but this change has no impact on log rotation implementation.  Here is an article that described the problem and the solution that I implemented.

[http://support.en.ctx.org.cn/ctx138064.citrix](http://support.en.ctx.org.cn/ctx138064.citrix)

@harikrishna-patnala can you comment if #861 is going to help in log rotation problem.

rhtyd commented:
tag:easypr

swill commented:
Can someone with a xenserver setup validate this one?  @bvbharat can you queue this one up for us?  Thanks...

bvbharatk commented:
### ACS CI BVT Run
 **Sumarry:**
 Build Number 103
 Hypervisor xenserver
 NetworkType Advanced
 Passed=71
 Failed=2
 Skipped=3

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**
* test_vpc_vpn.py

 * test_01_vpc_site2site_vpn Failed

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_routers.py
test_reset_vm_on_reboot.py
test_snapshots.py
test_deploy_vms_with_varied_deploymentplanners.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

SudharmaJain commented:
Previous CI failures were not relevant to my changes. 
tag:mergeready

cloudmonger commented:
### ACS CI BVT Run
 **Sumarry:**
 Build Number 422
 Hypervisor xenserver
 NetworkType Advanced
 Passed=105
 Failed=0
 Skipped=7

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**

**Skipped tests:**
test_01_test_vm_volume_snapshot
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_non_contigiousvlan.py
test_login.py
test_deploy_vm_iso.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_routers_network_ops.py
test_disk_offerings.py

jayakarteek commented:
Test results;
when I executed the setupxenserver.sh file , following is the output
iptables: Bad rule (does a matching rule exist in that chain?).
mv: invalid option -- n
Try `mv --help' for more information.
success

mv -n option is provided in line 52

SudharmaJain commented:
@jayakarteek Thanks for testing. I have made the respective code changes.

jayakarteek commented:
Tested the PUll Request.
Logs are rotated as expected in /var/log/cloud folder

SudharmaJain commented:
tag:mergeready

DaanHoogland commented:
@blueorangutan package

blueorangutan commented:
@DaanHoogland a Jenkins job has been kicked to build packages. I'll keep you posted as I make progress.

blueorangutan commented:
Packaging result: ✔centos6 ✔centos7 ✔debian. JID-738

DaanHoogland commented:
@blueorangutan test centos7 xenserver-65sp1

blueorangutan commented:
@DaanHoogland a Trillian-Jenkins test job (centos7 mgmt + xenserver-65sp1) has been kicked to run smoke tests

blueorangutan commented:
<b>Trillian test result (tid-1122)</b>
Environment: xenserver-65sp1 (x2), Advanced Networking with Mgmt server 7
Total time taken: 48439 seconds
Marvin logs: https://github.com/blueorangutan/acs-prs/releases/download/trillian/pr883-t1122-xenserver-65sp1.zip
Intermitten failure detected: /marvin/tests/smoke/test_privategw_acl.py
Intermitten failure detected: /marvin/tests/smoke/test_routers_network_ops.py
Intermitten failure detected: /marvin/tests/smoke/test_vpc_redundant.py
Test completed. 49 look ok, 2 have error(s)


Test | Result | Time (s) | Test File
--- | --- | --- | ---
test_05_rvpc_multi_tiers | `Failure` | 516.05 | test_vpc_redundant.py
test_01_create_redundant_VPC_2tiers_4VMs_4IPs_4PF_ACL | `Failure` | 603.72 | test_vpc_redundant.py
test_04_rvpc_privategw_static_routes | `Failure` | 919.67 | test_privategw_acl.py
test_01_vpc_site2site_vpn | Success | 341.27 | test_vpc_vpn.py
test_01_vpc_remote_access_vpn | Success | 176.81 | test_vpc_vpn.py
test_01_redundant_vpc_site2site_vpn | Success | 607.75 | test_vpc_vpn.py
test_02_VPC_default_routes | Success | 420.28 | test_vpc_router_nics.py
test_01_VPC_nics_after_destroy | Success | 764.05 | test_vpc_router_nics.py
test_04_rvpc_network_garbage_collector_nics | Success | 1649.08 | test_vpc_redundant.py
test_03_create_redundant_VPC_1tier_2VMs_2IPs_2PF_ACL_reboot_routers | Success | 875.47 | test_vpc_redundant.py
test_02_redundant_VPC_default_routes | Success | 1122.15 | test_vpc_redundant.py
test_09_delete_detached_volume | Success | 15.76 | test_volumes.py
test_08_resize_volume | Success | 95.91 | test_volumes.py
test_07_resize_fail | Success | 100.95 | test_volumes.py
test_06_download_detached_volume | Success | 20.35 | test_volumes.py
test_05_detach_volume | Success | 100.28 | test_volumes.py
test_04_delete_attached_volume | Success | 10.21 | test_volumes.py
test_03_download_attached_volume | Success | 15.27 | test_volumes.py
test_02_attach_volume | Success | 11.02 | test_volumes.py
test_01_create_volume | Success | 387.85 | test_volumes.py
test_change_service_offering_for_vm_with_snapshots | Success | 364.52 | test_vm_snapshots.py
test_03_delete_vm_snapshots | Success | 280.25 | test_vm_snapshots.py
test_02_revert_vm_snapshots | Success | 191.77 | test_vm_snapshots.py
test_01_create_vm_snapshots | Success | 105.82 | test_vm_snapshots.py
test_deploy_vm_multiple | Success | 353.38 | test_vm_life_cycle.py
test_deploy_vm | Success | 0.03 | test_vm_life_cycle.py
test_advZoneVirtualRouter | Success | 0.03 | test_vm_life_cycle.py
test_10_attachAndDetach_iso | Success | 32.49 | test_vm_life_cycle.py
test_09_expunge_vm | Success | 125.23 | test_vm_life_cycle.py
test_08_migrate_vm | Success | 66.12 | test_vm_life_cycle.py
test_07_restore_vm | Success | 0.11 | test_vm_life_cycle.py
test_06_destroy_vm | Success | 15.19 | test_vm_life_cycle.py
test_03_reboot_vm | Success | 20.23 | test_vm_life_cycle.py
test_02_start_vm | Success | 25.26 | test_vm_life_cycle.py
test_01_stop_vm | Success | 30.27 | test_vm_life_cycle.py
test_CreateTemplateWithDuplicateName | Success | 206.39 | test_templates.py
test_08_list_system_templates | Success | 0.03 | test_templates.py
test_07_list_public_templates | Success | 0.04 | test_templates.py
test_05_template_permissions | Success | 0.06 | test_templates.py
test_04_extract_template | Success | 5.16 | test_templates.py
test_03_delete_template | Success | 5.11 | test_templates.py
test_02_edit_template | Success | 90.10 | test_templates.py
test_01_create_template | Success | 105.79 | test_templates.py
test_10_destroy_cpvm | Success | 226.78 | test_ssvm.py
test_09_destroy_ssvm | Success | 199.18 | test_ssvm.py
test_08_reboot_cpvm | Success | 141.46 | test_ssvm.py
test_07_reboot_ssvm | Success | 143.92 | test_ssvm.py
test_06_stop_cpvm | Success | 342.06 | test_ssvm.py
test_05_stop_ssvm | Success | 164.09 | test_ssvm.py
test_04_cpvm_internals | Success | 1.22 | test_ssvm.py
test_03_ssvm_internals | Success | 3.54 | test_ssvm.py
test_02_list_cpvm_vm | Success | 0.15 | test_ssvm.py
test_01_list_sec_storage_vm | Success | 0.14 | test_ssvm.py
test_02_list_snapshots_with_removed_data_store | Success | 131.15 | test_snapshots.py
test_01_snapshot_root_disk | Success | 31.39 | test_snapshots.py
test_04_change_offering_small | Success | 96.07 | test_service_offerings.py
test_03_delete_service_offering | Success | 0.04 | test_service_offerings.py
test_02_edit_service_offering | Success | 0.06 | test_service_offerings.py
test_01_create_service_offering | Success | 0.09 | test_service_offerings.py
test_02_sys_template_ready | Success | 0.16 | test_secondary_storage.py
test_01_sys_vm_start | Success | 0.19 | test_secondary_storage.py
test_01_scale_vm | Success | 5.20 | test_scale_vm.py
test_09_reboot_router | Success | 70.57 | test_routers.py
test_08_start_router | Success | 65.47 | test_routers.py
test_07_stop_router | Success | 20.22 | test_routers.py
test_06_router_advanced | Success | 0.06 | test_routers.py
test_05_router_basic | Success | 0.04 | test_routers.py
test_04_restart_network_wo_cleanup | Success | 5.52 | test_routers.py
test_03_restart_network_cleanup | Success | 141.00 | test_routers.py
test_02_router_internal_adv | Success | 0.85 | test_routers.py
test_01_router_internal_basic | Success | 0.55 | test_routers.py
test_router_dns_guestipquery | Success | 48.92 | test_router_dns.py
test_router_dns_externalipquery | Success | 0.07 | test_router_dns.py
test_router_dhcphosts | Success | 102.52 | test_router_dhcphosts.py
test_router_dhcp_opts | Success | 31.29 | test_router_dhcphosts.py
test_01_updatevolumedetail | Success | 5.12 | test_resource_detail.py
test_01_reset_vm_on_reboot | Success | 50.47 | test_reset_vm_on_reboot.py
test_createRegion | Success | 0.04 | test_regions.py
test_create_pvlan_network | Success | 5.23 | test_pvlan.py
test_dedicatePublicIpRange | Success | 0.43 | test_public_ip_range.py
test_03_vpc_privategw_restart_vpc_cleanup | Success | 958.82 | test_privategw_acl.py
test_02_vpc_privategw_static_routes | Success | 748.15 | test_privategw_acl.py
test_01_vpc_privategw_acl | Success | 172.74 | test_privategw_acl.py
test_03_migration_options_storage_tags | Success | 39.35 | test_primary_storage.py
test_02_edit_primary_storage_tags | Success | 0.12 | test_primary_storage.py
test_01_primary_storage_nfs | Success | 39.11 | test_primary_storage.py
test_01_primary_storage_iscsi | Success | 79.71 | test_primary_storage.py
test_01_deploy_vms_storage_tags | Success | 31.34 | test_primary_storage.py
test_createPortablePublicIPRange | Success | 15.20 | test_portable_publicip.py
test_createPortablePublicIPAcquire | Success | 15.51 | test_portable_publicip.py
test_isolate_network_password_server | Success | 36.36 | test_password_server.py
test_UpdateStorageOverProvisioningFactor | Success | 0.13 | test_over_provisioning.py
test_oobm_zchange_password | Success | 30.76 | test_outofbandmanagement.py
test_oobm_multiple_mgmt_server_ownership | Success | 16.44 | test_outofbandmanagement.py
test_oobm_issue_power_status | Success | 10.30 | test_outofbandmanagement.py
test_oobm_issue_power_soft | Success | 15.43 | test_outofbandmanagement.py
test_oobm_issue_power_reset | Success | 15.34 | test_outofbandmanagement.py
test_oobm_issue_power_on | Success | 15.39 | test_outofbandmanagement.py
test_oobm_issue_power_off | Success | 15.36 | test_outofbandmanagement.py
test_oobm_issue_power_cycle | Success | 15.36 | test_outofbandmanagement.py
test_oobm_enabledisable_across_clusterzones | Success | 87.81 | test_outofbandmanagement.py
test_oobm_enable_feature_valid | Success | 5.17 | test_outofbandmanagement.py
test_oobm_enable_feature_invalid | Success | 5.14 | test_outofbandmanagement.py
test_oobm_disable_feature_valid | Success | 5.19 | test_outofbandmanagement.py
test_oobm_disable_feature_invalid | Success | 0.11 | test_outofbandmanagement.py
test_oobm_configure_invalid_driver | Success | 0.10 | test_outofbandmanagement.py
test_oobm_configure_default_driver | Success | 0.09 | test_outofbandmanagement.py
test_oobm_background_powerstate_sync | Success | 23.41 | test_outofbandmanagement.py
test_extendPhysicalNetworkVlan | Success | 15.35 | test_non_contigiousvlan.py
test_01_nic | Success | 650.80 | test_nic.py
test_releaseIP | Success | 268.23 | test_network.py
test_reboot_router | Success | 594.89 | test_network.py
test_public_ip_user_account | Success | 10.26 | test_network.py
test_public_ip_admin_account | Success | 40.31 | test_network.py
test_network_rules_acquired_public_ip_3_Load_Balancer_Rule | Success | 86.73 | test_network.py
test_network_rules_acquired_public_ip_2_nat_rule | Success | 76.62 | test_network.py
test_network_rules_acquired_public_ip_1_static_nat_rule | Success | 100.73 | test_network.py
test_delete_account | Success | 343.29 | test_network.py
test_02_port_fwd_on_non_src_nat | Success | 70.76 | test_network.py
test_01_port_fwd_on_src_nat | Success | 83.92 | test_network.py
test_nic_secondaryip_add_remove | Success | 247.98 | test_multipleips_per_nic.py
test_list_zones_metrics | Success | 0.24 | test_metrics_api.py
test_list_volumes_metrics | Success | 5.49 | test_metrics_api.py
test_list_vms_metrics | Success | 126.46 | test_metrics_api.py
test_list_pstorage_metrics | Success | 0.33 | test_metrics_api.py
test_list_infrastructure_metrics | Success | 0.46 | test_metrics_api.py
test_list_hosts_metrics | Success | 0.37 | test_metrics_api.py
test_list_clusters_metrics | Success | 0.38 | test_metrics_api.py
login_test_saml_user | Success | 19.22 | test_login.py
test_assign_and_removal_lb | Success | 148.95 | test_loadbalance.py
test_02_create_lb_rule_non_nat | Success | 207.58 | test_loadbalance.py
test_01_create_lb_rule_src_nat | Success | 208.62 | test_loadbalance.py
test_03_list_snapshots | Success | 0.06 | test_list_ids_parameter.py
test_02_list_templates | Success | 0.04 | test_list_ids_parameter.py
test_01_list_volumes | Success | 0.03 | test_list_ids_parameter.py
test_07_list_default_iso | Success | 0.06 | test_iso.py
test_05_iso_permissions | Success | 0.11 | test_iso.py
test_04_extract_Iso | Success | 5.17 | test_iso.py
test_03_delete_iso | Success | 95.17 | test_iso.py
test_02_edit_iso | Success | 0.09 | test_iso.py
test_01_create_iso | Success | 21.11 | test_iso.py
test_04_rvpc_internallb_haproxy_stats_on_all_interfaces | Success | 510.01 | test_internal_lb.py
test_03_vpc_internallb_haproxy_stats_on_all_interfaces | Success | 334.50 | test_internal_lb.py
test_02_internallb_roundrobin_1RVPC_3VM_HTTP_port80 | Success | 1064.93 | test_internal_lb.py
test_01_internallb_roundrobin_1VPC_3VM_HTTP_port80 | Success | 721.57 | test_internal_lb.py
test_dedicateGuestVlanRange | Success | 10.31 | test_guest_vlan_range.py
test_UpdateConfigParamWithScope | Success | 0.17 | test_global_settings.py
test_rolepermission_lifecycle_update | Success | 6.22 | test_dynamicroles.py
test_rolepermission_lifecycle_list | Success | 6.02 | test_dynamicroles.py
test_rolepermission_lifecycle_delete | Success | 5.90 | test_dynamicroles.py
test_rolepermission_lifecycle_create | Success | 5.95 | test_dynamicroles.py
test_rolepermission_lifecycle_concurrent_updates | Success | 6.00 | test_dynamicroles.py
test_role_lifecycle_update_role_inuse | Success | 5.94 | test_dynamicroles.py
test_role_lifecycle_update | Success | 10.99 | test_dynamicroles.py
test_role_lifecycle_list | Success | 5.90 | test_dynamicroles.py
test_role_lifecycle_delete | Success | 10.99 | test_dynamicroles.py
test_role_lifecycle_create | Success | 5.92 | test_dynamicroles.py
test_role_inuse_deletion | Success | 5.90 | test_dynamicroles.py
test_role_account_acls_multiple_mgmt_servers | Success | 8.34 | test_dynamicroles.py
test_role_account_acls | Success | 8.33 | test_dynamicroles.py
test_default_role_deletion | Success | 6.01 | test_dynamicroles.py
test_04_create_fat_type_disk_offering | Success | 0.08 | test_disk_offerings.py
test_03_delete_disk_offering | Success | 0.04 | test_disk_offerings.py
test_02_edit_disk_offering | Success | 0.06 | test_disk_offerings.py
test_02_create_sparse_type_disk_offering | Success | 0.10 | test_disk_offerings.py
test_01_create_disk_offering | Success | 0.08 | test_disk_offerings.py
test_deployvm_userdispersing | Success | 60.91 | test_deploy_vms_with_varied_deploymentplanners.py
test_deployvm_userconcentrated | Success | 131.36 | test_deploy_vms_with_varied_deploymentplanners.py
test_deployvm_firstfit | Success | 236.80 | test_deploy_vms_with_varied_deploymentplanners.py
test_deployvm_userdata_post | Success | 30.53 | test_deploy_vm_with_userdata.py
test_deployvm_userdata | Success | 146.26 | test_deploy_vm_with_userdata.py
test_02_deploy_vm_root_resize | Success | 0.07 | test_deploy_vm_root_resize.py
test_01_deploy_vm_root_resize | Success | 0.07 | test_deploy_vm_root_resize.py
test_00_deploy_vm_root_resize | Success | 221.96 | test_deploy_vm_root_resize.py
test_deploy_vm_from_iso | Success | 368.61 | test_deploy_vm_iso.py
test_DeployVmAntiAffinityGroup | Success | 171.64 | test_affinity_groups.py
test_06_copy_template | Skipped | 0.00 | test_templates.py
test_static_role_account_acls | Skipped | 0.02 | test_staticroles.py
test_11_ss_nfs_version_on_ssvm | Skipped | 0.02 | test_ssvm.py
test_nested_virtualization_vmware | Skipped | 0.00 | test_nested_virtualization.py
test_06_copy_iso | Skipped | 0.00 | test_iso.py
test_06_verify_guest_lspci_again | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_05_change_vm_ostype_restart | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_04_verify_guest_lspci | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_03_verify_libvirt_attach_disk | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_02_verify_libvirt_after_restart | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_01_verify_libvirt | Skipped | 0.00 | test_deploy_virtio_scsi_vm.py
test_deploy_vgpu_enabled_vm | Skipped | 0.03 | test_deploy_vgpu_enabled_vm.py
test_3d_gpu_support | Skipped | 0.04 | test_deploy_vgpu_enabled_vm.py

rhtyd commented:
Merging based on CI results, no regressions seen and manual test confirmation on the PR.
